rules/sonic-psud.mk

```diff
@@ -4,7 +4,7 @@
 
 SONIC_PSUD_PY2 = sonic_psud-1.0-py2-none-any.whl
 $(SONIC_PSUD_PY2)_SRC_PATH = $(SRC_PATH)/sonic-platform-daemons/sonic-psud
-$(SONIC_PSUD_PY2)_DEPENDS = $(SONIC_PY_COMMON_PY2)
+$(SONIC_PSUD_PY2)_DEPENDS = $(SONIC_PY_COMMON_PY2) $(SONIC_PLATFORM_COMMON_PY2)
```

Comment:
Why we still have PY2?

Reply:
Because some vendors have not yet ported their platform APIs to Python 3. :( We are allowing them until the next release to do so.